analyze_text.py

```diff
@@ -1,4 +1,12 @@
 # Remember to reach out for help after your own due diligence
 
 def analyze_text(text):
-    # Your code here
+    total_char = len(text)
+    num_e = 0
+    mystr = "e"
+    for char in text:
+        if char == "e" or "E":
```

Comment:
An `or` statement in an `if` statement separates out the two comparisons. So currently, the computer is evaluating you're if statement as `if 'char' is equal to' e' OR if 'E'`, rather than `if 'char' is equal to 'e' OR if 'char' is equal to 'E'`. when using`or` within an `if` statement, you need to explicitly state what you want to compare on both sides of the `or`